starter-code/views/index.hbs

```diff
@@ -1,3 +1,5 @@
-<div>
-
+<div class="home">
+    <img src="/images/beer.png" alt="beer">
+    <button id="beers">Check the Beers!</button>
+    <button id="random-beer">Check a Random Beer!</button>
```

Comment:
I think you should hyperlinks with your buttons like <a href="/beers"><button id="beers">Check the Beers!</button></a> and <a href="/random-beer"><button id="random-beer">Check a Random Beer!</button></a> next time